### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by g33gs** [projectdiscovery/subfinder#516] [Q&A] [answered]
Installing through 'go install' presents error message:

CMD RUN -- `go install -v github.com/projectdiscovery/subfinder/cmd/subfinder@latest`

OUTPUT -- `go install: github.com/projectdiscovery/subfinder/cmd/subfinder@latest: module github.com/projectdiscovery/subfinder@latest found (v2.5.0+incompatible), but does not contain package github.com/projectdiscovery/subfinder/cmd/subfinder`

Searched for fixes from past issues doing the same thing, but those didn't appear to resolve this.

**Comment by ehsandeep** [comment, marked as answer]
@phoxanon there is missing `v2` in the installation cmd:

```
go install -v github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest
```
